### ISSUE TITLE
validation loss not changing

### ISSUE THREAD

**Issue by abinramapuram** [ialhashim/DenseDepth#70]
I have created models by training on NYU, KITTI and synthetic data set. For all the training I got much different values for training loss and validation loss. Training loss decreases after each epochs and the validation loss has no significant change after each epochs. Same trend has seen in three data sets I have tried. Can you interpret it??

**Comment by ialhashim** [comment]
The model is not learning. The most obvious thing to check is ranges and zero values.
